packages/bot/src/transformers/invite.ts

```diff
@@ -1,15 +1,16 @@
-import type { DiscordApplication, DiscordInviteCreate } from '@discordeno/types'
-import type { Application, Bot, User } from '../index.js'
+import type { BigString, DiscordApplication, DiscordInviteMetadata } from '@discordeno/types'
+import type { Application, Bot, ScheduledEvent, User } from '../index.js'
+import type { InviteStageInstance } from './stageInviteInstace.js'
 
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-export function transformInvite(bot: Bot, payload: DiscordInviteCreate) {
+export function transformInvite(bot: Bot, payload: DiscordInviteMetadata) {
   const props = bot.transformers.desiredProperties.invite
   const invite = {} as Invite
 
-  if (props.channelId && payload.channel_id) invite.channelId = bot.transformers.snowflake(payload.channel_id)
+  if (props.channelId && payload.channel && payload.channel.id) invite.channelId = bot.transformers.snowflake(payload.channel.id)
```

Comment:
```suggestion
  if (props.channelId && payload.channel?.id) invite.channelId = bot.transformers.snowflake(payload.channel.id)
```

Reply:
What happened to the channel_id prop? Is that never going to be provided anymore?